containers.go

```diff
@@ -86,6 +86,14 @@ type ContainerStore interface {
 	// with the specified ID.
 	SetNames(id string, names []string) error
 
+	// AddNames adds the list of names associated with a container to storage with the
+	// supplied values.
```

Comment:
I am not a native speaker but something similar to the comment below easier to brain parse:
```suggestion
	// AddNames adds the supplied values to the list of names associated with the container with the specified id.
```
Same at other locations.